packages/dds/merge-tree/src/localReference.ts

```diff
@@ -354,7 +349,7 @@ export class LocalReferenceCollection {
 
 			lref.link(this.segment, offset, atRefs.push(lref).last);
 
-			if (refHasRangeLabels(lref) || refHasTileLabels(lref)) {
+			if (refHasTileLabels(lref)) {
 				this.hierRefCount++;
```

Comment:
I kept `hierRefCount` and the tiles stuff around as I wasn't sure if this safe to remove.